src/main/java/edu/kis/vh/nursery/dataStructures/DataStructure.java

```diff
@@ -0,0 +1,16 @@
+package edu.kis.vh.nursery.dataStructures;
+
+public interface DataStructure {
```

Comment:
Names of operations do not correspond to operating the data structure, thus do not reflect the interface name.